### PR TITLE
feat: adding devcontainer config

### PR DESCRIPTION
**Pull request by pauldotyu**
Adding Dev Container config and "Open in Codespaces" button at the top of the README.

### CODE REVIEW

**Comment by rbtr** [comment]
+ this change fixes the CI issue here https://github.com/microsoft/retina/pull/84

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Retina
 
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=746962176)
```
Comment (rbtr) [inline README.md:3]: this is great 😍

**Comment by pauldotyu** [comment]
> Can you add a couple of lines about `Testing Done`?

Sure thing. I was able to install this in a local KIND cluster using the following command but let me do some additional testing and document.

```
IMAGE_NAMESPACE=microsoft/retina make helm-install
```

File: .devcontainer/installMoreTools.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Install the required tools and dependencies
+sudo apt-get update && sudo apt-get install lsb-release wget software-properties-common gnupg -y
+
```
[the remark this answers is not in the document]
Comment (pauldotyu) [reply .devcontainer/installMoreTools.sh:5]: I've added these additional tools

**Review by rbtr — APPROVED** [review]
lgtm, thanks!

**Pull request merged**